api/models.py

```diff
@@ -155,14 +155,18 @@ def user_name(self):
 class Reservation(models.Model):
     attendee_number = models.PositiveIntegerField()
     created_on = models.DateTimeField(auto_now_add=True)
-    observations = models.TextField()
+    observations = models.TextField(blank=True)
     paid_amount = models.DecimalField(
         max_digits=10,
         decimal_places=2,
         validators=[MinValueValidator(Decimal('0.00'))]
     )
     user = models.ForeignKey('users.wineuser', on_delete=models.PROTECT)
     event_occurrence = models.ForeignKey(EventOccurrence, on_delete=models.PROTECT)
+    status = models.IntegerField(
+        choices=RESERVATION_STATUS,
+        default=RESERVATION_CONFIRMED,
```

Comment:
Shouldn't the default value be "RESERVATION_CREATED"?
Btw: how would be "confirm" reservations? Does this involve and additional workflow for wineries?

Reply:
In theory, it should, but we don't have that additional workflow to change the status to confirmed or rejected, that should be done using the payment info. Maybe we should think of how to mock the payments first.